src/System.Private.CoreLib/shared/System/Diagnostics/Tracing/CounterGroup.cs

```diff
@@ -52,11 +52,10 @@ private void OnEventSourceCommand(object? sender, EventCommandEventArgs e)
         {
             if (e.Command == EventCommand.Enable || e.Command == EventCommand.Update)
             {
-                string valueStr;
                 float value;
                 Debug.Assert(e.Arguments != null);
 
-                if (e.Arguments.TryGetValue("EventCounterIntervalSec", out valueStr) && float.TryParse(valueStr, out value))
+                if (e.Arguments.TryGetValue("EventCounterIntervalSec", out string? valueStr) && float.TryParse(valueStr, out value))
```

Comment:
Nit: Given you're in-lining valueStr's declaration, would be more consistent with `value` in-lined as well.

Reply:
Good point.